### DISCUSSION TITLE
Grafana Loki sink Label Expansion issue with helm chart

### DISCUSSION THREAD

**Discussion by Amikuto** [vectordotdev/helm-charts#380] [Q&A] [answered]
Hello!

I'm trying to deploy Vector via helm chart. 
I need to add some labels to Loki. According to your [documentation](https://vector.dev/docs/reference/configuration/sinks/loki/#label-expansion) I need to add the following value to values.yaml file:
```yaml
labels: 
    k8s_container_name: {{ kubernetes.container_name  }}
```
But this configuration crashes with the help error, Helm does not see the kubernetes.container_name function.
I found the following [solution](https://github.com/helm/helm/issues/2798) to the problem.
```yaml
k8s_container_name: '"{{ "{{" }} kubernetes.container_name {{ "}}" }}"'
```

However, now the following value is in the config map:
```yaml
k8s_pod_namespace: '"{{ kubernetes.pod_namespace }}"'
```

Unfortunately, with a single quote, vector does not see this configuration and I do not receive any logs at all

Could you help me please?

**Comment by jszwedko** [comment]
I believe:

```
k8s_container_name: "{{ "{{" }} kubernetes.container_name {{ "}}" }}"
```

Should work. I would check what the config map ends up being rendered with.

**Comment by Amikuto** [comment, marked as answer]
Solution that works for me

```yaml
"k8s_pod_name": "{{`{{kubernetes.k8s_pod_name}}`}}"
```
